addons/stock_account/models/stock_quant.py

```diff
@@ -21,7 +21,7 @@ def _compute_value(self):
         a estimation more than a real value).
         """
         for quant in self:
-            if not quant.location_id._should_be_valued() or\
+            if quant.location_id and not quant.location_id._should_be_valued() or\
```

Comment:
really not sure here
location_id is required so this test seems useless? isn't it missing a required in the view?

Reply:
missing location_id owner_id company_id in the depends

Reply:
``` python
        for quant in self:
            # If the user didn't enter a location yet while enconding a quant.
            if not quant.location_id:
                quant.value = 0
                return

            if not quant.location_id._should_be_valued...
```